scripts/pylint.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

[nit] (preferring the most portable shebang)

Reply:
ACK - fixed